tools/stub/ssv-node-stub.go

```diff
@@ -19,29 +22,52 @@ func StartStub(port uint) {
 
 	router := chi.NewMux()
 
+	stop := make(chan bool, 1)
 	s := stub{
 		keypair: kp,
 		port:    port,
 		router:  router,
+		stop:    stop,
 	}
 
 	api.BindSSVApi(router, s)
 
-	s.Start()
+	go func() {
+		s.Start()
+	}()
+
+	return func() {
+		stop <- true
```

Comment:
```suggestion
		close(stop)
```
so that it can be read as many time as you want in as many go routines as needed